### ISSUE TITLE
Incomplete loading with iOS mobile safari

### ISSUE THREAD

**Issue by calvintwr** [linkedin/dustjs#509]
Please refer to this stackoverflow issue for full details: http://stackoverflow.com/questions/26352839/linkedin-dustjs-on-node-express-ios-safari

This may not be purely a dust issue, but I am unable to pinpoint where the fault it. Effectively, I have removed all modules and revert the app back to a standard Express setup. The problem persists with dust, but is gone when i replaced the view engine with EJS.

In short, on the first clean GET request, page loading could not be completed when using iOS Safari. Inspector is showing that latter resources are not getting a response. They, will receive the response after 2 minutes.

There seem to be a limit as to the page size before this happens, meaning to say, as I include more js or images after the threshold, they will run into "not getting a response".
